src/components/accordion.ts

```diff
@@ -0,0 +1,78 @@
+import {Component, View, NgClass, OnDestroy} from 'angular2/angular2';
+
+@Component({
+  selector: 'ngb-accordion, [ngb-accordion]',
+  properties: ['onlyOneOpen: closeOthers'],
+  host: {'class': 'panel-group'}
+})
+@View({template: `<ng-content></ng-content>`})
+export class NgbAccordion {
+  private onlyOneOpen: boolean;
+  private groups: Array<NgbAccordionGroup> = [];
+
+  addGroup(group: NgbAccordionGroup): void { this.groups.push(group); }
+
+  closeOthers(openGroup): void {
+    if (!this.onlyOneOpen) {
+      return;
+    }
+
+    this.groups.forEach((group: NgbAccordionGroup) => {
+      if (group !== openGroup) {
+        group.isOpen = false;
+      }
+    });
+  }
+
+  removeGroup(group: NgbAccordionGroup): void {
+    const index = this.groups.indexOf(group);
+    if (index !== -1) {
+      this.groups.splice(index, 1);
+    }
+  }
+}
+
+@Component({selector: 'ngb-accordion-group, [ngb-accordion-group]', properties: ['heading', 'isOpen', 'isDisabled']})
+@View({
+  template: `
```

Comment:
In difference with the dropdown, I prefer to have multilines templates starting on a new line, easier to read. Thoughts?